app/views/catalog/_home_text.html.erb

```diff
@@ -1,14 +1,9 @@
 <div class="welcome-message">
-    <h2>Welcome to the beta Princeton University Library catalog.</h2>
-    <p>This catalog is being developed to eventually replace our twenty year old <%= t('blacklight.voyager') %> and incorporate the best features of new types of library search technology. Please note that the <%= t('blacklight.voyager') %> and the Books+ systems will continue to be available throughout our beta development phase.</p>
-
-    <p>This site is in active development. For more details about these changes, please see our <%= link_to 'release notes', page_path('notes') %>.</p>
-
-    <p>We need your feedback! Please <a href="http://library.princeton.edu/help/contact-us" target="_blank">send us any questions or comments</a>.</p>
-    <p>This catalog is based on <a href="http://projectblacklight.org/" target="_blank">Blacklight technology</a>, which is being developed by University libraries around the world.</p>
+    <h2>Welcome to the Princeton University Library Catalog.</h2>
+    <p>This catalog is currently in under development and will eventually replace  our twenty year old <%= t('blacklight.voyager') %> later this summer. Please note that the <%= t('blacklight.voyager') %> and the Books+ systems will continue to be available throughout our development phase. <%= link_to 'Learn more', page_path('about') %>.</p>
 </div>
-<div class="other-resources">
-    <h3>Other resources</h3>
+<div>
```

Comment:
Does it look less weird with the border on the left? <img width="493" alt="screen shot 2016-04-28 at 10 43 04 am" src="https://cloud.githubusercontent.com/assets/8161144/14889754/0b7fcb4a-0d2e-11e6-90a7-42bcfd60a216.png">